[IMP] pos/terminals: update adyen page (#18138)
odoo/documentation: content/applications/sales/point_of_sale/payment_methods/terminals.rst

Payment terminals
=================

> Connect and integrate a payment terminal with a :ref:`POS system <pos/use/open-register>` to accept

Reviewer: Maybe I'm missing something, but I don't get why we're linking to the 'Open the POS register' section. Maybe to the 'Create a POS' section?

(Also, you might want to rebase already now, since that page was updated recently.)

Author: You are right, that actually does not make sense

multiple payment options, including credit and debit cards.

.. _pos/terminals/configuration:

Configuration
=============

> To activate a payment terminal and allow processing payments through it, follow these steps:

Reviewer: Suggestion:

```suggestion
To activate a payment terminal and allow payments to be processed through it, follow these steps:
```

#. Go to the :ref:`POS settings <pos/use/settings>`.
#. Scroll down to the :guilabel:`Payment Terminals` section.
#. Enable the relevant payment terminal.
#. Click :guilabel:`Save`.
#. Click :icon:`fa-arrow-right` :guilabel:`Payment method` below the selected terminal to
   :doc:`configure the related payment method <../payment_methods>`.
#. On the payment method form:

>    #. Set the :guilabel:`Journal` type to :guilabel:`Bank`.

Reviewer 2: If you create a payment method that way, the Journal field is automatically set to Bank
Maybe just 
```suggestion
   #. Ensure the :guilabel:`Journal` type is set to :guilabel:`Bank`.
``` 
?

Author: You are right. I will adapt this 👍

>    #. In the :guilabel:`Integrations` field, select :guilabel:`Terminal` from the

Reviewer: ```suggestion
   #. In the :guilabel:`Integration` field, select :guilabel:`Terminal` from the
```

      :icon:`fa-caret-down` dropdown menu.
   #. In the :guilabel:`Integrate with` field, select one of the enabled terminals or click
      :guilabel:`Setup` on the terminal's card.

      .. tip::

>          It is also possible to :guilabel:`Activate` a new terminal directly from the terminal card.

Reviewer: I'm confused here. What do we mean by 'enabled terminals'? The only options I see are 'Activate' and 'Setup' (with Setup appearing after I have clicked 'Activate' in a trial DB ... in runbots, I never see any still with the 'Activate' button). And there doesn't seem to be any other way to 'select' a terminal. Could it be that you can only 'select' a terminal that has already been fully set up (i.e., where the terminal-specific config is already done)?

So, I don't understand the step or the tip. Maybe you do and can help me understand?

Author: You can select it from the drop-down menu (once you activate it, no need to configure it to select it here) or click the Setup button. And if it is not activated, you can enable it by clicking the Activate button.

<img width="715" height="689" alt="Image" src="https://github.com/user-attachments/assets/67a4db1d-7c7a-46d8-a793-410900acf0c5" />

Reviewer: I had totally missed that there was even a dropdown (I don't like that in Odoo you often don't even see that there's a dropdown if you don't hover over it. Yeah, it's 'neater' than indicating the dropdown with a line and icon, but especially in situations like this, you can miss it!)

Maybe we can say something like 'select one of the enabled terminals from the dropdown or, to XYZ, click Setup ...'

Let's discuss!

   #. Complete the terminal-specific configuration for one of the following providers:

      - :doc:`Adyen <terminals/adyen>`
      - :doc:`Dpopay <terminals/dpo>`
      - :doc:`Ingenico <terminals/ingenico>`
      - :doc:`Mercado Pago <terminals/mercado_pago>`
      - :doc:`Mollie <terminals/mollie>`
      - :doc:`Pine Labs <terminals/pine_labs>`
      - :doc:`QFPay <terminals/qfpay>`
      - :doc:`Razorpay <terminals/razorpay>`
      - :doc:`SIX <terminals/six>`
      - :doc:`Stripe <terminals/stripe>`
      - :doc:`Tyro <terminals/tyro>`
      - :doc:`Viva.com <terminals/viva_com>`
      - :doc:`Worldline <terminals/worldline>`

.. _pos/terminals/terminal-use:

Terminal use
============

To process an order :ref:`payment <pos/use/sell>` using a :ref:`configured terminal
<pos/terminals/configuration>` in the :ref:`POS register <pos/use/open-register>`, select the
terminal's :doc:`payment method <../payment_methods>` on the :guilabel:`Payment` screen, then follow

> the instructions on the terminal device.

Reviewer: ```suggestion
terminal's :doc:`payment method <../payment_methods>` on the :guilabel:`Payment` screen, then follow
the instructions displayed on the terminal screen.
```

Just to match earlier wording

Once the transaction is successful, the payment is automatically validated in the POS register.

.. note::
   - If a connection issue occurs between Odoo and the payment terminal, the transaction is
     automatically canceled. You can also manually abort and validate the order by clicking

>      :guilabel:`Force done`.

Reviewer: Is this in the POS register or on the terminal screen?
Also, should this maybe be two separate bullets? (not fully understand the use cases)

Author: It is in Odoo's POS register.
Sure 2 bullet points make sense

Reviewer: Should we make it clear that it's in the POS register?
